src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -272,7 +272,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     Computes any files that need to be copied to the build output folder for .NET Core.
     ============================================================
     -->
-  <UsingTask TaskName="EmbedAppNameInHost" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+  <UsingTask TaskName="PrepareAppHost" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
```

Comment:
👍 on name change, as this will also make more sense for additional 3.0 changes.

Reply:
Yep, this name has been bugging me for a while. :)

Reply:
nit: Although "Prepare" doesn't really mean too much. And the file is done after this target right? and it will just be copied around

Reply:
I can't think of anything better right now - if you have a better name, I'll be happy to change it.

Reply:
CreateAppHost might be good since it matches what we called it in the C# helper. I'm OK with Prepare too though.